testkit/src/main/scala/org/bitcoins/testkit/core/gen/NumberGenerator.scala

```diff
@@ -87,6 +87,9 @@ trait NumberGenerator {
   def scriptNumbers: Gen[ScriptNumber] =
     Gen.choose(Int64.min.toLong, Int64.max.toLong).map(ScriptNumber(_))
 
+  def timeLockScriptNumbers: Gen[ScriptNumber] =
+    Gen.choose(1L, UInt32.max.toLong - 1L).map(ScriptNumber(_))
```

Comment:
This seems artificially limited, IIRC `ScriptNumber` can be 5 bytes? Is this the consensus valid range that `ScriptNumbers` can be in for time lock operations? If so let's just add a quick scaladoc to the top of the method to indicate that.